src/renderer/bootstrap.tsx

```diff
@@ -50,17 +51,26 @@ export async function bootstrap(App: AppComponent) {
   await attachChromeDebugger();
   rootElem.classList.toggle("is-mac", isMac);
 
-  extensionLoader.init();
-  extensionDiscovery.init();
+  ExtensionLoader.getInstanceOrCreate().init();
+  ExtensionDiscovery.getInstanceOrCreate().init();
+
+  const userStore = UserStore.getInstanceOrCreate();
+  const clusterStore = ClusterStore.getInstanceOrCreate();
+  const extensionsStore = ExtensionsStore.getInstanceOrCreate();
+  const filesystemStore = FilesystemProvisionerStore.getInstanceOrCreate();
+  const themeStore = ThemeStore.getInstanceOrCreate();
+  const hotbarStore = HotbarStore.getInstanceOrCreate();
+  const helmRepoManager = HelmRepoManager.getInstanceOrCreate();
 
   // preload common stores
   await Promise.all([
     userStore.load(),
     hotbarStore.load(),
     clusterStore.load(),
     extensionsStore.load(),
-    filesystemProvisionerStore.load(),
+    filesystemStore.load(),
     themeStore.init(),
+    helmRepoManager.init(),
```

Comment:
@Nokel81  was there a reason for this? Now renderer boot is very slow (blank screen visible for few seconds).

/cc @jim-docker

Reply:
No, we can move that line to outside the array